music21/expressions.py

```diff
@@ -647,11 +866,26 @@ class WholeStepMordent(Mordent):
     >>> m.direction
     'down'
     >>> m.size
-    <music21.interval.Interval M2>
+    <music21.interval.Interval M-2>
     '''
-    def __init__(self, **keywords):
+    def __init__(self, **keywords) -> None:
+        # no accidental supported here, just "WholeStep"
+        kw = keywords.copy()
+        if 'accid' in kw:
+            del kw['accid']
```

Comment:
this seems like a place to raise an exception rather than silently deleting it.  Better to fail fast rather than letting a bug slip through.

Reply:
Sounds good to me.

Reply:
Done.